packages/overlay/test/restore-focus.test.js

```diff
@@ -34,71 +34,23 @@ customElements.define(
 describe('restore focus', () => {
   let wrapper, overlay, focusable, focusInput;
 
-  before(() => {
-    focusInput = document.createElement('input');
-    document.body.appendChild(focusInput);
-  });
-
-  after(() => {
-    document.body.removeChild(focusInput);
-  });
-
   beforeEach(() => {
+    focusInput = document.createElement('input');
```

Comment:
**note:** I'll planning to check the `focus-ring` attribute's presence in #5839, so it's better to reset the focus node before each test.